rest/taskana-rest-spring/src/main/java/pro/taskana/rest/RestConfiguration.java

```diff
@@ -32,11 +39,14 @@
 @Configuration
 @ComponentScan
 @EnableTransactionManagement
-public class RestConfiguration {
+public class RestConfiguration extends WebMvcConfigurationSupport {
```

Comment:
extending WebMvcConfigurationSupport  doesn't allow me to run taskana-rest-spring-example correctly

Reply:
What problems are caused by it when you try to run it?

Reply:
I usually run the taskana-spring-example running the main class of the project, I we extends webMvcConfigurationSupport The application doesn´t work when I try to open it in the browser.